runtime/syntax/lsp_markdown.vim

```diff
@@ -4,7 +4,8 @@
 " URL:		http://neovim.io
 " Remark:	Uses markdown syntax file
 
-runtime! syntax/markdown.vim
+let md = fnamemodify("$VIMRUNTIME/syntax/lsp_markdown.vim", ":p:h") . "/markdown.vim"
+execute "runtime! " . md
```

Comment:
I think you can use `<sfile>` for this: 

```suggestion
execute 'source' expand('<sfile>:p:h') .. '/markdown.vim'
```

Not sure if appending `'/markdown.vim'` works on Windows since it uses `\` as a path separator

Reply:
Awesome, just tested it and it all works as expected. Thanks!

I'm also not sure if this works on windows... If anyone else know, let me know!